code-challenges.test.js

```diff
@@ -49,4 +121,29 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+// FAIL  week-5/week-5-assessment-jricks86/code-challenges.test.js
+// ● fullHouse › returns true if the array is a full house
+
+//   ReferenceError: fullHouse is not defined
+
 // b) Create the function that makes the test pass.
+// input: array of 5 numbers
+// output: boolean to return if full house is exactly one pair and one three of a kind.
+
+const fullHouse = (array) => {
+    // create empty object to hold number of counts of each number
+  const counts = {};
+//   loop through array and add all the numbers to the counts object
+  for (let i = 0; i < array.length; i++) {
+    counts[array[i]] = counts[array[i]] ? counts[array[i]] + 1 : 1;
+  }
+//   new variable to check to see if the number of times a number appears is equal to the number of times a pair appears
+  const values = Object.values(counts);
```

Comment:
This pseudo code does not explain what is happening in the code well. I suggest making sure you understand the concepts here and break each thing down to get a full understanding.